e2e/cypress/support/commands.ts

```diff
@@ -139,105 +139,48 @@ Cypress.Commands.add('generateAdminToken', (idToken: string) => {
   });
 });
 
-Cypress.Commands.add('cleanupPoliciesByIDPrefix', (idToken: string, idPrefix: string) => {
-  cy.request({
-    auth: { bearer: idToken },
-    method: 'GET',
-    url: '/apis/iam/v2beta/policies',
-    failOnStatusCode: false
-  }).then((resp) => {
-    const body = resp.body;
-    for (const policy of body.policies) {
-      if (policy.id.startsWith(idPrefix)) {
-        cy.request({
-          auth: { bearer: idToken },
-          method: 'DELETE',
-          url: `/apis/iam/v2beta/policies/${policy.id}`,
-          failOnStatusCode: false
-        });
-      }
-    }
-  });
-});
+Cypress.Commands.add('cleanupV2IAMObjectsByIDPrefixes',
+  (idToken: string, idPrefix: string, objectPlurals: string[]) => {
```

Comment:
Consider calling it `iamObjects` rather than `objectPlurals`; the latter does not really provide any info to the reader.

Reply:
Disagree that mentioning you want the plural version doesn't provide any value. Don't have a strong opinion though so can change.